src/ChooseYourFramework.ts

```diff
@@ -50,6 +51,12 @@ export class ChooseYourFramework extends EightBittr {
      */
     public readonly groupHolder: GroupHoldr<ActorGroups>;
 
+    /**
+     * Cache-based wrapper around localStorage.
+     */
+    public readonly itemsHolder: ItemsHoldr<
+    Record<string, boolean>>;
```

Comment:
This reminds me, I need to set up CI to catch formatting issues. And commit hooks. #22.